frontend/packages/pipelines-plugin/src/components/quicksearch/pipeline-quicksearch-utils.ts

```diff
@@ -123,7 +137,7 @@ export const createTask = (url: string, namespace: string) => {
         ...task.metadata.annotations,
         [TektonTaskAnnotation.installedFrom]: TEKTONHUB,
       };
-      await k8sCreate(TaskModel, task);
+      await k8sCreate(task.apiVersion === 'tekton.dev/v1' ? TaskModel : TaskModelV1Beta1, task);
```

Comment:
I think return is better then await here. Wdty?

Reply:
Agreed